packages/shared/src/utils/url/url.ts

```diff
@@ -35,6 +40,23 @@ export const urlForLanguage = (lang: string, url: string = window.location.href)
     return `${current_url}`;
 };
 
+export const urlSetQuery = (queryObj: TQueryObj, url: string = window.location.href) => {
+    const current_url = new URL(url);
+
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
@sergei-deriv Please do not ignore `eslint` errors, Instead try to handle them 🙏🏻 
You can use `forEach` instead of `for of`.

Reply:
@farzin-deriv Ok, I'll try to change it